2.12.0

artur-trzesiok opened the pull request:
pako -> fflate (faster performance of snapshots/compression_level>0)
introducing logarithmic_depth_buffer (True by default)
fullscreen js api
shadow_map_update fix
fix orbit controller on snapshots
fix low zbuffer accuracy for software rendering screenshots
trackball/orbit controller are now mobile friendly
mesh volume, texture data, volume dynamic update fix
fix 3d/3dSpecular k3d.points z-buffer
dependencies bump (threejs)
removing misleading documentation
camera_fov argument of k3d.plot init function has no effect #316
dat.gui -> lil-gui 
axesHelper dynamic change fix 
respect plot.grid_auto_fit = False when plot.render()

CagtayFabry commented:
There are some great fixes in this one 👍 🚀 

any plans on when you will release this new version @artur-trzesiok ?

artur-trzesiok commented:
> There are some great fixes in this one 👍 🚀
> 
> any plans on when you will release this new version @artur-trzesiok ?

Thank you for the kind words. I changed here like...a lot. Tests are green but we didn't cover with tests top-right GUI that I changed here. I will do QA session today with it. If in next 5 hours everything will be ok I will do a relese :)
